### DISCUSSION TITLE
在VIVO和OPPO的一些联发科CPU的手机上，发现了PAG的一些崩溃。

### DISCUSSION THREAD

**Discussion by coolstar1204** [Tencent/libpag#2316] [General] [answered]
pag版本，4.1.49和4.2.100都上线出现了。
![image](https://github.com/Tencent/libpag/assets/2889783/fa209382-aa5c-44ce-9494-75a9ed877bc3)
![image](https://github.com/Tencent/libpag/assets/2889783/f6de0344-7d81-4eb7-b875-b198691fe81c)

**Comment by kevingpqi123** [comment, marked as answer]
这个问题是 MTK 芯片兼容性的问题，已经有其它业务反馈过，根据我们排查的信息，其它使用 OpenGL 相关方法的业务也普遍存在该问题，暂时没有解决方案
（1）由于一直没有复现路径，这里不太好处理，如果你们有稳定的复现路径，欢迎反馈过来我们可以进一步分析该问题；
（2）对于 pag 而言，如果不涉及全屏播放、文本编辑、占位图替换，可以切换至 PAGImageView，可以大幅降低这种概率
PAGImageView 的相关介绍可以阅读这篇文章：https://pag.art/docs/use-pagimageview.html
